exspy/misc/eels/gosh_gos.py

```diff
@@ -79,20 +87,25 @@ class GoshGOS(TabulatedGOS):
         "subshell_factor": None,
     }
 
-    def __init__(self, element_subshell, gos_file_path=None):
+    def __init__(self, element_subshell, gos_file_path=None, source="dft"):
         """
         Parameters
         ----------
         element_subshell : str
             For example, 'Ti_L3' for the GOS of the titanium L3 subshell
         gos_file_path : str
             The path of the gosh file to use.
+        source : str
+            The source of the GOS data. Options are 'dft' or 'dirac'.
         """
 
         if gos_file_path is None:
+            source = source.lower()
+            assert source in GOSH_SOURCES.keys(), f"Invalid source: {source}"
+            self._name = f"{source}_gosh"
```

Comment:
Does it need the "gosh" suffix?
```suggestion
            self._name = source
```